govtech/detectors/cves/cve_2020_3452/src/main/java/com/google/tsunami/plugins/detectors/cves/cve_2020_3452/CVE20203452Detector.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2020 GovTech
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.tsunami.plugins.detectors.cves.cve_2020_3452;
```

Comment:
Can you please follow [Google Java coding style](https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names) and avoid using underscore in package names? Thanks!

Reply:
Done!

Reply:
Done!